block.go

```diff
@@ -249,19 +265,27 @@ func (b *EthBlock) ResolveLink(p []string) (*node.Link, []string, error) {
 	return nil, nil, fmt.Errorf("resolved item was not a link")
 }
 
+// Size returns the size in bytes of the serialized object
 func (b *EthBlock) Size() (uint64, error) {
 	// TODO:
```

Comment:
In case you're wondering what this should return... the answer is it doesn't matter. This function will go away as a part of the great go-ipld-format refactor (every IPLD format handles size differently so it's kind of useless at the moment).